### PR TITLE
Postage Board MCU

### PR DESCRIPTION
**Pull request by yanfali**
Adding this to repo, because it's got some undocumented pins.

## Description
The maker did not submit this to QMK firmware. I am doing so on behalf of a user.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/PostageBoard/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    QMKURL
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [_BASE] = LAYOUT(
+        KC_A,    KC_1,    MO(_FN)
+    ),
+    [_FN] = LAYOUT(
+        QMKBEST, QMKURL,  RESET
```
Comment (zvecr) [inline keyboards/handwired/PostageBoard/keymaps/default/keymap.c]: `RESET` cant be tapped as you already have MO held down, OSL?

Comment (yanfali) [reply keyboards/handwired/PostageBoard/keymaps/default/keymap.c]: lol good catch

File: keyboards/handwired/PostageBoard/PostageBoard.c [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "PostageBoard.h"
```
Comment (fauxpark) [inline keyboards/handwired/PostageBoard/PostageBoard.c]: Keyboard name should be lowercase ;)

File: keyboards/handwired/PostageBoard/keymaps/default/config.h [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// place overrides here
```
Comment (fauxpark) [inline keyboards/handwired/PostageBoard/keymaps/default/config.h]: This file can be deleted.

File: keyboards/handwired/PostageBoard/PostageBoard.c (hunk 2 above)
Comment (yanfali) [reply keyboards/handwired/PostageBoard/PostageBoard.c]: good point

File: keyboards/handwired/postageboard/readme.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+However, neither the Teensy, nor the Pro Micro are designed for keyboards specifically. They are just general Arduino-like boards meant for a variety of DIY projects. The Postage Board is designed for mechanical keyboards specifically, and thus the process of assembling your keyboard is easier, and the final product is more refined.
+
+* Keyboard Maintainer: [LifeIsOnTheWire](https://github.com/LifeIsOnTheWire/Postage-Board)
+* Hardware Supported: Available at GB
+* Hardware Availability: Available at GB
+
+Make example for this keyboard (after setting up your build environment):
+
+    make PostageBoard:default
```
Comment (fauxpark) [inline keyboards/handwired/postageboard/readme.md]:
```suggestion
    make postageboard:default
```

File: keyboards/handwired/postageboard/readme.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+However, neither the Teensy, nor the Pro Micro are designed for keyboards specifically. They are just general Arduino-like boards meant for a variety of DIY projects. The Postage Board is designed for mechanical keyboards specifically, and thus the process of assembling your keyboard is easier, and the final product is more refined.
+
+* Keyboard Maintainer: [LifeIsOnTheWire](https://github.com/LifeIsOnTheWire/Postage-Board)
+* Hardware Supported: Available at GB
+* Hardware Availability: Available at GB
+
+Make example for this keyboard (after setting up your build environment):
+
+`    make handwired/postageboard/r1:default`
```
Comment (fauxpark) [inline keyboards/handwired/postageboard/readme.md]:
```suggestion
    make handwired/postageboard/r1:default
```

Don't need the backticks here.

File: keyboards/handwired/postageboard/readme.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+or
+
+`    make handwired/postageboard/mini:default`
```
Comment (fauxpark) [inline keyboards/handwired/postageboard/readme.md]:
```suggestion
    make handwired/postageboard/mini:default
```

**Pull request merged**